dist/init/linux-systemd/README.md

```diff
@@ -6,26 +6,153 @@ if you have any questions. Feel free to prepend to your question
 the username of whoever touched the file most recently, for example
 `@wmark re systemd: …`.
 
-The provided file should work with systemd version 219 or later. It might work with earlier versions.
+The provided file should work with systemd version 229 or later.
 The easiest way to check your systemd version is to look at the version of the installed package
-(e.g. 'sudo yum info systemd' on RedHat/Fedora systems).
-
-## Instructions
+(e.g. 'sudo dpkg --status systemd | grep Version' on Debian/Ubuntu or 'sudo yum info systemd' on RedHat/Fedora systems).
 
 We will assume the following:
 
-* that you want to run caddy as user `www-data` and group `www-data`, with UID and GID 33
+* that you want to run caddy as user `www-data` and group `www-data`
 * you are working from a non-root user account that can use 'sudo' to execute commands as root
 
 Adjust as necessary or according to your preferences.
 
-First, put the caddy binary in the system wide binary directory and give it
-appropriate ownership and permissions:
+## Install
+
+Install Caddy to `/usr/local/bin`:
 
 ```bash
-sudo cp /path/to/caddy /usr/local/bin
-sudo chown root:root /usr/local/bin/caddy
-sudo chmod 755 /usr/local/bin/caddy
+# Note: change amd64 to 386, arm5, arm6, or arm7 as necessary,
+#       also you can add plugins as desired
+curl --fail --silent --show-error --location \
+  'https://caddyserver.com/download/linux/amd64?plugins=' --output /tmp/caddy.tar.gz
+
+mkdir -p /tmp/caddy-package/
+tar xvf /tmp/caddy.tar.gz -C /tmp/caddy-package/
+sudo chmod 755 /tmp/caddy-package/caddy
+sudo chown root:root /tmp/caddy-package/caddy
+sudo mv /tmp/caddy-package/caddy /usr/local/bin/
+```
+
+Then install `/etc/systemd/system/caddy.service` and `/etc/tmpfiles.d/caddy.conf`:
+
+```
+curl --fail --silent --show-error --location --remote-name-all \
+  https://raw.githubusercontent.com/mholt/caddy/master/dist/init/linux-systemd/caddy.service
+curl --fail --silent --show-error --location --remote-name-all \
+  https://raw.githubusercontent.com/mholt/caddy/master/dist/init/linux-systemd/caddy.conf
+
+sudo chown root:www-data caddy.service
+sudo chown root:www-data caddy.conf
+
+sudo chmod 0755 caddy.service
+sudo chmod 0755 caddy.conf
```

Comment:
Above four `chmod` should've already been covered by `--no-same-owner` and Caddy's buildsrv packaging the files correctly (with correct permissions). System-wide service files should be `root:root` in any case, and don't need the *execute-bit*.

(I know the old instructions had these lines. This is about the now needlessly changed ownership and permissions.)